src/Message/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Message';
```

Comment:
The component is not exported from the library. Additional export should be added in the `/src/index.js`

Reply:
Done.